app/controllers/application_controller/automate.rb

```diff
@@ -142,10 +142,10 @@ def resolve
   def build_results
     options = {
       :vmdb_object => @sb[:obj],
-      :fqclass     => @resolve[:new][:starting_object],
-      :message     => @resolve[:new][:object_message]
+      :fqclass     => @resolve[:new][:starting_object]
     }
     @resolve[:state_attributes] = {} if params[:button] == 'throw'
+    automation_attrs["message"] = @resolve[:new][:object_message]
```

Comment:
@d-m-u In this line automation_attrs has not been initialized as a hash. Can you move this line to after line 149 when the automation_attrs is a hash